### PR TITLE
Add support for multiple apps

### PR DESCRIPTION
**Pull request by sh4nks**
Hey!

This PR should make it possible to use Flask-Babel with more apps. I have also added a basic test.
I also added support for wheels and did some clean up.

This is kind of related to #107 and #112.

Let me know if there is anything else I should change.

### CODE REVIEW

**Comment by sh4nks** [comment]
Btw, is there a specific reason why the delimiter for the translation directories is a semi-colon (;)?

File: flask_babel/__init__.py [hunk 1]
```diff
@@ -54,36 +54,32 @@ class Babel(object):
...
-            self.init_app(app)
+            self.init_app(app, **kwargs)
 
-    def init_app(self, app):
-        """Set up this instance for use with *app*, if no app was passed to
-        the constructor.
+    def init_app(self, app, default_locale='en', default_timezone='UTC',
+                 default_domain='messages', date_formats=None,
+                 configure_jinja=True):
+        """Sets up the Flask-BabelEx extension.
```
Comment (TkTech) [inline flask_babel/__init__.py]: `flask-babel`

File: flask_babel/__init__.py [hunk 2]
```diff
@@ -118,6 +117,23 @@ def init_app(self, app):
                 newstyle=True
             )
 
+    def get_app(self, reference_app=None):
+        """Helper method that implements the logic to look up an a
+        pplication.
```
Comment (TkTech) [inline flask_babel/__init__.py]: Typo (a pplication/application)

File: tests/tests.py [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
+import flask
+from babel.support import NullTranslations
+
 import flask_babel as babel
 from flask_babel import gettext, ngettext, lazy_gettext, get_translations
-from babel.support import NullTranslations
 from flask_babel._compat import text_type
 
+#sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+
```
Comment (TkTech) [inline tests/tests.py]: ?

**Comment by sh4nks** [comment]
Fixed and rebased :)

**Comment by b0g3r** [comment]
@TkTech may be you merge this pr?

**Comment by b0g3r** [comment]
@sh4nks How about adding support for blueprints?

**Pull request closed without merging**